src/ts/state/reducer.ts

```diff
@@ -57,10 +57,10 @@ import { ExpandRanges, IFilterOptions } from "../createTyperighterPlugin";
 import { getBlocksFromDocument, nodeContainsText } from "../utils/prosemirror";
 import { Node } from "prosemirror-model";
 import {
-  selectSingleBlockInFlightById,
-  selectBlockQueriesInFlightForSet,
+  selectSingleBlockInRequestInFlightById,
+  selectRequestInFlightById,
   selectMatchByMatchId,
-  selectBlockQueriesInFlightById
+  selectBlocksInFlightById as selectBlocksInFlightById
```

Comment:
I can't tell if my eyes are deceiving me here, but the alias looks to be the same as the original?

Reply:
Heh, well spotted – missed this artefact left by VSCode refactoring. Should've waited to merge! I'll follow up.